lib/createInlinePluginCreator.js

```diff
@@ -188,7 +143,7 @@ function createInlinePluginCreator(packages, multiContext, synchronizer, flags)
 			await waitFor("_readyToGenerateNotes", pkg);
 
 			// Update pkg deps.
-			updateManifestDeps(pkg, path);
+			updateManifestDeps(pkg);
```

Comment:
shouldn't this happen in the prepare step? i.e., right now our API usage in dry-mode leaves the repository in a dirty state with file changes.

Reply:
Yep. Added `dryRun` mode. https://github.com/dhoulb/multi-semantic-release/pull/45/commits/b3fa9e95f5790a5c5cde86b8fc521d1d728cf448